internal/engine/watchmanager.go

```diff
@@ -203,7 +203,12 @@ func (w *WatchManager) OnChange(ctx context.Context, st store.RStore) {
 			continue
 		}
 
-		watcher, err := w.fsWatcherMaker(target.Dependencies(), ignore, logger)
+		deps := target.Dependencies()
```

Comment:
i wonder if this filtering should happen further up, like in watchableTargetsForManifest?